src/main/java/com/scylladb/jmx/utils/APIBuilder.java

```diff
@@ -116,23 +113,20 @@ public boolean contains(final ObjectName name) {
 
         @Override
         public Integer getCount() {
-            lock.readLock().lock();
-            try {
+            synchronized(lock) {
                 return wrapped.getCount() + tableMBeans.size();
-            } finally {
-                lock.readLock().unlock();
             }
         }
 
         @Override
         public void addMBean(final DynamicMBean bean, final ObjectName name, final RegistrationContext ctx)
                 throws InstanceAlreadyExistsException {
+            System.out.println(name);
```

Comment:
Is this left on purpose?

Reply:
No. It should not be there. Too bad avi already merged.